### ISSUE TITLE
Issue: "[X.range.value]: must be a finite number"

### ISSUE THREAD

**Issue by Maedie** [EthanJWright/foundryvtt-importer#93]
**Describe the bug**
When attempting to parse the following statblock, I receive two model validation errors: "[FeatData.range.value]: must be a finite number" and "[WeaponData.range.value]: must be a finite number". When parsed, neither the reaction "Life Transfer" nor the ranged weapon attack "Necrotic Curse/Words of Ice" appear in the statblock. I believe the issue is with those two items.

**To Reproduce**

```
Once-Elias, Cruelty of the Ancient 
Medium Humanoid, Neutral Evil
Armor Class 18 (magic)
Hit Points 430
Speed 30 ft.
STR 12 (+1) DEX 16 (+3) CON 18 (+4) INT 20 (+5) WIS 22 (+6) CHA 20 (+5)
Saving Throws Intelligence +9, Wisdom +10, Charisma +9
Skills Persuasion +9
Damage Resistances cold, necrotic, bludgeoning, piercing and slashing from non-magical attacks
Senses passive Perception 16
Languages Common, Elvish, Sylvan
Proficiency Bonus +4, Initiative +8
TRAITS
Indoctrinated Creso. When Once-Elias is reduced to 0 hit points and is not healed within 1 minute, his body is destroyed. He reforms in the Antipathy within the next 1d4 days.
Of Two Souls. Each round, Once-Elias switches between Elias' abilities and Enmity's abilities, referred to as his first and second form. Any ability that Once-Elias concentrates on remains in effect until he loses concentration. 
Focused. Once-Elias has advantage on Concentration saving throws.
ACTIONS
Psychic Shortsword/Bludgeoning Wave. Melee Weapon Attack: +10 to hit, reach 5 ft., two targets. Hit: 14 (2d8+5) psychic or bludgeoning damage.
Necrotic Curse/Words of Ice. Ranged Weapon Attack: +10 to hit, range 30 ft., two targets. Hit: 14 (1d10+9) necrotic or cold damage.
The Once Divine (Elias, 1/1, C). Once-Elias attempts to charm three creatures within 120 ft. of him. All targets must make a DC 18 Wisdom saving throw. On a failure, the target is charmed. While charmed in this way, a creature sees Once-Elias as comparable to the divine, and will do anything within their power to satisfy his whims. The target may make another DC 18 Wisdom saving throw at the end of each turn to attempt to break free from the enchantment. 
The Once Defeated (Enmity, 1/1, C). Once-Elias attempts to manipulate the minds of three creatures within 120 ft. of him. All targets must make a DC 17 Intelligence saving throw. On a failure, the target believes that they are out of air and are suffocating. The target can survive for a number of rounds equal to their Constitution modifier, with a minimum of one round. At the start of their next turn, the target drops to 0 hit points and is dying. The target may make another DC 17 Intelligence saving throw at the end of each turn to attempt to break free from the enchantment. 
BONUS ACTIONS
Show Restraint (Elias, recharge 4-6). Once-Elias targets a creature within 30 ft. of him and attempts to force them to restrain themselves. The target must make a DC 18 Wisdom saving throw. On a failure, the target is restrained until the end of their next turn.
Lost in Tides (Enmity, recharge 4-6). Once-Elias targets a creature within 30 ft. of him and attempts to manipulate their mind into a riptide. The target must make a DC 17 Intelligence saving throw. On a failure, the target is prone until the end of their next turn.
REACTIONS
Life Transfer (5/5). When Once-Elias is hit with a melee or ranged attack within 30 ft. of him, he can force the attacker to make a DC 16 Constitution saving throw. On a failure, the attacker takes 10 (1d10+5) necrotic damage. 
```

**Expected behaviour**
I expected the reaction "Life Transfer" to exist on the statblock as a reaction and expected "Necrotic Curse/Words of Ice" to appear next to the melee attack in the attacks section of the statblock.

**Screenshots**
![ErrorStatblock](https://user-images.githubusercontent.com/91714442/231674198-7e47f3f3-b2fb-4101-87b0-6a8604dcdbd4.png)
![ErrorMessage](https://user-images.githubusercontent.com/91714442/231674208-eb2bb0e1-1678-4fd0-834e-fb8aa875e2e2.png)

**Additional context**
I'm unsure how to format abilities such as Ranged Weapon Attacks, Melee/Ranged Spell Attacks, and abilities that require saving throws and/or deal damage on a failure, in a way where the content can be reliably parsed. I'd like to have some sort of template for these use-cases, as I couldn't find them covered in the Actors 5e section of the module's README. 
Thank you so much for creating this module, as well! It's absolutely fantastic, I want to be able to use it to its full potential.

**Comment by EthanJWright** [comment]
Awesome thanks for posting, I'll take a look at this when I get a chance - the item parsing is kind of sketch and brute force, but I try to make sure it works for all example data I come across, so this helps strengthen it!

**Comment by Maedie** [comment]
No worries! Let me know if I can provide anything else for you, I'm more than happy to help.

**Comment by EthanJWright** [comment]
Should be patched in version #94 v0.2.43

Thanks for submitting, issues like this are the only way I quash bugs =)
Open another issue if anything else comes up, or this didn't work, thanks!

**Comment by Maedie** [comment]
The issue appears patched! Thank you so much.
